app.js

```diff
@@ -4,9 +4,19 @@ define(function(require) {
 		monster = require('monster'),
 		timezone = require('monster-timezone');
 
+	var appSubmodules = [
+		'newAccountWizard'
```

Comment:
I would argue for now you can just use `wizard` as a submodule name to help declutter the methods' prefix.

We can easily update it later and make it more specific if this app needs a wizard to create different entities than accounts (not likely).